lib/rollbar/request_data_extractor.rb

```diff
@@ -28,7 +28,12 @@ def extract_request_data_from_rack(env)
       session = rollbar_filtered_params(sensitive_params, rollbar_request_session(rack_req))
       route_params = rollbar_filtered_params(sensitive_params, rollbar_route_params(env))
 
-      params = request_params.merge(get_params).merge(post_params).merge(raw_body_params)
+      params =
+        if !raw_body_params.empty?
```

Comment:
Hi @elight we cannot change that logic in the way you changed it. You are not adding the GET params to the `params` variable.

For `request_params` and `post_params` we may need to change some logic. But this cannot be merged with this change. Probably tests should be changed to fit the new behavior, but this implementation cannot be released with this change.

If you want to investigate what's happening in those you are welcome, if you don't a free slot to do that just notice us and we'll investigate it.

Thanks!